package.json

```diff
@@ -53,6 +57,7 @@
     "eslint": "8.7.0",
     "eslint-plugin-react-hooks": "4.3.0",
     "gulp": "~4.0.2",
+    "tailwindcss": "^3.3.3",
```

Comment:
is this a dev or regular dependency (I actually see some others like eslint that should be devDependencies instead)